components/brave_new_tab_ui/components/default/footer/togetherTooltip/index.tsx

```diff
@@ -25,7 +25,7 @@ const TogetherTooltip: React.FunctionComponent<Props> = function (props) {
         <S.Body>
           {getLocale('togetherPromptDescription')}
         </S.Body>
-        <LinkButton href='https://together.brave.com/widget'>
```

Comment:
How about making this as a shared constants?
With that, we could use it for https://github.com/brave/brave-core/pull/9549 also.

Reply:
@stephendonner if we pull from constants, I think you can add to https://github.com/brave/brave-core/blob/master/components/brave_new_tab_ui/constants/new_tab_ui.ts